packages/flutter/test/material/data_table_test.dart

```diff
@@ -1496,4 +1496,75 @@ void main() {
       const Offset(width - borderVertical, height - borderHorizontal),
     );
   });
+
+  testWidgets('DataTable set interior border test',
+          (WidgetTester tester) async {
```

Comment:
nit: Please fix the formatting here: These two lines should just be one line and then the body of the method should just be indented by two spaces.

Reply:
Hi @goderbauer , I accidentally have removed the fork that is linked to the PR.
Is it possible to fetch the PR and link it to a new  fork, so the PR can be updated. Or how should I update the this PR.